docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md

```diff
@@ -0,0 +1,23 @@
+---
+slug: "new-referencevalue-parameter-create-payment-endpoint"
+title: "New referenceValue parameter in the Create Payment endpoint request body"
+createdAt: 2023-10-24T00:00:00.000Z
+hidden: false
+type: "added"
+excerpt: "New parameter available in the Create Payment endpoint request body."
+---
+The `referenceValue` parameter was added to the Create Payment endpoint. This new field is mandatory and must be sent on the endpoint to represent the payment amount without interest applied. The values ​​of the `value` and `referenceValue` fields will be equal whenever the value of the `installmentsInterestRate` field is equal to zero. See the examples below:
```

Comment:
Please add a link to the endpoint reference.

Reply:
Adicionado.